### ISSUE TITLE
Graphic artifacts on fonts in ui

### ISSUE THREAD

**Issue by scompo** [atom/atom#17311]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Graphic artifacts displaying fonts.

![artifacts](https://user-images.githubusercontent.com/2161625/39957631-63fa6218-55f6-11e8-915b-dc42584bec7b.png)

### Steps to Reproduce

1. Turn it on.

**Expected behavior:** Just the text rendered correctly

**Actual behavior:** Graphics artifacts on rendered text (see image).

**Reproduces how often:** 100%

### Versions

output from `atom --version:`
```
Atom    : 1.24.0
Electron: 1.6.16
Chrome  : 56.0.2924.87
Node    : 7.4.0

```
Output fom `apm --version`
```
apm  1.18.12
npm  3.10.10
node 6.9.5 x64
atom 1.24.0
python 2.7.15rc1
git 2.17.0
```
Ubuntu 18.04
4.15.0-20-generic

### Additional Information

Just turn it on.

**Comment by rsese** [comment]
Thanks for the report!

> Atom    : 1.24.0

I think this might be a duplicate of https://github.com/atom/atom/issues/15737 which was fixed in 1.25 - please update to the latest release version of Atom (currently 1.26.1) and let us know if you still see the issue.

---

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).

**Comment by scompo** [comment]
Yup, it's resolved in 1.26.0, thank you!
